scripts/sensorshow

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python3
+
+'''
+    Script to show Voltage and Current Sensor status.
+'''
+from tabulate import tabulate
+from natsort import natsorted
+import argparse
+import os
+import sys
+
+# mock the redis for unit test purposes #
+try:
+    if os.environ["UTILITIES_UNIT_TESTING"] == "1":
+        modules_path = os.path.join(os.path.dirname(__file__), "..")
+        test_path = os.path.join(modules_path, "tests")
+        libs_path = os.path.join(test_path, "mocked_libs")
+        sys.path.insert(0, modules_path)
+        sys.path.insert(0, test_path)
+        sys.path.insert(0, libs_path)
+        import mock_tables.dbconnector
+except KeyError:
+    pass
+
+from sonic_platform_base.sensor_base import CurrentSensorBase
+from sonic_platform_base.sensor_base import VoltageSensorBase
+from swsscommon.swsscommon import SonicV2Connector
+
+header = ['Sensor', '', 'High TH', 'Low TH', 'Crit High TH', 'Crit Low TH', 'Warning', 'Timestamp']
+
+TIMESTAMP_FIELD_NAME = 'timestamp'
+HIGH_THRESH_FIELD_NAME = 'high_threshold'
+LOW_THRESH_FIELD_NAME = 'low_threshold'
+CRIT_HIGH_THRESH_FIELD_NAME = 'critical_high_threshold'
+CRIT_LOW_THRESH_FIELD_NAME = 'critical_low_threshold'
+WARNING_STATUS_FIELD_NAME = 'warning_status'
+VOLTAGE_INFO_TABLE_NAME = 'VOLTAGE_INFO'
+CURRENT_INFO_TABLE_NAME = 'CURRENT_INFO'
+
+
+class SensorShow(object):
+    def __init__(self, type):
+        self.db = SonicV2Connector(use_unix_socket_path=True)
+        self.db.connect(self.db.STATE_DB)
+        self.field_name = type
+        if type == "voltage":
+            self.table_name = VOLTAGE_INFO_TABLE_NAME
+            header[1]  = 'Voltage({})'.format(VoltageSensorBase.get_unit())
```

Comment:
I suppose user might override get_unit, right?

Reply:
If we let user override, then we would have to display the unit on every line of the CLI o/p - which might not look clean. Let me know if you would prefer this.  Alternately we could remove the get_unit API and specify the sensor data to be provided in a specific unit. (currently other CLIs are also assuming a specific unit e.g the temperature o/p is in Celsius).

Reply:
Fixed